go/pkg/flags/flags.go

```diff
@@ -64,6 +68,12 @@ var (
 	StartupCapabilities = flag.Bool("startup_capabilities", true, "Whether to self-configure based on remote server capabilities on startup.")
 	// RPCTimeouts stores the per-RPC timeout values.
 	RPCTimeouts map[string]string
+	// KeepAliveTime specifies gRPCs keepalive time parameter.
+	KeepAliveTime = flag.Duration("grpc_keepalive_time", 0*time.Second, "After a duration of this time if the client doesn't see any activity it pings the server to see if the transport is still alive. If zero or not set, the mechanism is off.")
```

Comment:
Is the default value set like that to emphasize the unit? If so, I think that would be more useful in the help text where it would displayed to the user and the developer can still read it here.

Also, worth noting that the default it 0.